vl-kontrakt-fp-risk/src/main/java/no/nav/foreldrepenger/kontrakter/risk/v1/HentRisikovurderingDto.java

```diff
@@ -4,5 +4,5 @@
 
 import jakarta.validation.constraints.NotNull;
 
-public record HentRisikovurderingDto(@NotNull UUID konsumentId) {
-}
+// TODO Sett @NotNull på saksnummer når fpsak og fprisk tar i bruk ny kontrakt
```

Comment:
Tenker du her å fjerne NotNull på konsumentId ?

Reply:
Hvis vi har en behandlinguuid så trengs ikke saksnummer annet enn ved lagring.
Men du tenker kanskje på å søke på saksnummer hvis det ikke finnes noe for oppgitt behandlingUuid?

Reply:
Ja, jeg tenkte egentlig at fprisk, bør kunne returnerer seneste risikovurdering på et saksnummer. Men når jeg tenker etter er dette kanskje dumt? Fprisk vet jo ikke om behandlingen er henlagt i fpsak. Kanskje denne logikken for å finne seneste risikovurdering bør holdes i fpsak?

Reply:
Tja - det at en førstegangsbehandling er henlagt påvirker vel ikke risikoklassifiseringen, men det er mindre sannsynlig at risiko er vurdert av saksbehandler. Sak vet hvilke behandleringer som er behandlet ferdig eller henlagt, Risk vet hvilke som er vurdert - så begge er potensielle kilder.

Har vi sett noe på forekomst av flere vurderinger / sak ? 
* Utsatt oppstart mer enn noen uker vil gi ny førstegang - det er 10-20 tilfelle / uke
* flere mønstre?

Det er slett ikke uvanlig at noen sender ny 1gangsøknad før gjeldende er behandlet - men da er de sjelden vurdert.